*** ./lazykali.sh	2016-05-23 12:14:33.244747393 +0200
--- /usr/bin/lazykali	2016-05-23 12:23:36.320765479 +0200
***************
*** 1065,1080 ****
  }
  
  ### Update Exploitdb
  function updateexploitdb {
! 	echo -e "\033[31mThis script will update your Exploitdb\033[m"
! 	cd /usr/share/exploitdb
! 	rm -rf archive.tar.bz2
! 	wget http://www.exploit-db.com/archive.tar.bz2
! 	tar xvfj archive.tar.bz2
! 	rm -rf archive.tar.bz2
! 	echo -e "\e[32m[-] Done Updating Exploitdb!\e[0m"	
  }
  
  #### Searchsploit
  function searchsploit {
  	echo -e "\033[31mWhat do you want to Hack Today?\033[m"
--- 1065,1093 ----
  }
  
  ### Update Exploitdb
+ #function updateexploitdb {
+ #	echo -e "\033[31mThis script will update your Exploitdb\033[m"
+ #	cd /usr/share/exploitdb
+ #	rm -rf archive.tar.bz2
+ #	wget http://www.exploit-db.com/archive.tar.bz2
+ #	tar xvfj archive.tar.bz2
+ #	rm -rf archive.tar.bz2
+ #	echo -e "\e[32m[-] Done Updating Exploitdb!\e[0m"	
+ #}
+ 
  function updateexploitdb {
!     echo -e "\033[31mThis script will update your Exploitdb\033[m"
!     cd /usr/share/exploitdb
!     wget https://github.com/offensive-security/exploit-database/archive/master.zip
!     unzip -qq master.zip
!     rsync -a exploit-database-master/ /usr/share/exploitdb/
!     rm -rf master.zip exploit-database-master/
!     echo -e "\e[32m[-] Done Updating Exploitdb!\e[0m"
  }
  
+ 
+ 
+ 
  #### Searchsploit
  function searchsploit {
  	echo -e "\033[31mWhat do you want to Hack Today?\033[m"
